src/components.js

```diff
@@ -32,6 +32,10 @@ export { SubscriptionRenewView } from "./Components/SubscriptionRenew/Subscripti
 export { PaymentSuccessModal } from "./Components/PaymentSuccess/PaymentSuccessModal";
 export { PaymentSuccessView } from "./Components/PaymentSuccess/PaymentSuccessView";
 export { NewsletterWithHook as NewsLetter } from "./Components/NewsLetter/NewsLetter";
+export { NewsletterUpdateModal } from "./Components/NewsletterUpdate/NewsletterUpdateModal";
```

Comment:
Isn't the newsletter list missing from the exports? Double check for others.

Reply:
yup it's missing, will export it